src/v/cluster/rm_stm.h

```diff
@@ -144,6 +144,7 @@ class rm_stm final : public persisted_stm {
 
     static constexpr int8_t prepare_control_record_version{0};
     static constexpr int8_t fence_control_record_version{0};
+    static constexpr int8_t tx_seq_constrol_record_version{0};
```

Comment:
nit: Typo in variable name "control". I usually find cspell plugin in vscode handy, it highlights typos as I write the code/comments, fwiw.